src/pipeline.cc

```diff
@@ -544,7 +546,10 @@ class PipelineWorker : public Nan::AsyncWorker {
         image = image.embed(baton->extendLeft, baton->extendTop, baton->width, baton->height,
           VImage::option()->set("extend", VIPS_EXTEND_BACKGROUND)->set("background", background));
       }
-
+      // Median - must happen before blurring, due to the utility of blurring after thresholding
+      if (shouldApplyMedian) {
+        image = sharp::Median(image, baton->medianSize);
```

Comment:
This can probably use libvips' `median` operation, something like:
```cc
if (baton->medianSize > 0) {
  image = image.median(baton->medianSize);
}
```

Reply:
Done, although I am missing something... do we still need 

``` 
 VImage Median(VImage image, int const m) {
      return image.rank(m, m, (m*m)/2);
}
```
at https://github.com/lovell/sharp/pull/1161/files/1cbeabbed071bc6ad7543e2c80bd5989f5e07221#diff-3ad302ad0bfc04ecfaaf7c27e0b92174R362 ?